static/app/views/sharedGroupDetails/sharedGroupHeader.tsx

```diff
@@ -33,6 +41,27 @@ function SharedGroupHeader({group}: Props) {
               />
             )}
           </ShortIdWrapper>
+
+          <TimeStamp data-test-id="sgh-timestamp">
+            {'Last seen '}
+            {(event.dateCreated ?? event.dateReceived) && (
+              <EventTimeLabel>
+                {getDynamicText({
+                  fixed: 'Jan 1, 12:00 AM',
+                  value: (
+                    <Tooltip
+                      isHoverable
+                      showUnderline
+                      title={<EventCreatedTooltip event={event} />}
+                      overlayStyle={{maxWidth: 300}}
+                    >
+                      <DateTime date={date_obj.toLocaleString()} />
```

Comment:
I think we can avoid lines 22 and 23 by passing in the raw date time string into the `<DateTime>` component like this:

```
<DateTime
  date={
    group.latestEvent?.dateReceived ??
    group.latestEvent?.dateCreated
  }
/>
```

Also nit: I saw that we prefer dateCreated over dateReceived in the issue details page ([here](https://github.com/getsentry/sentry/blob/469ad6afb896aa6e67e1bd58a6fe54959e1386b2/static/app/views/issueDetails/groupEventCarousel.tsx#L417)), should we maybe switch the order just to keep things consistent?